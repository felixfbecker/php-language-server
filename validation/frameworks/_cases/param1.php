<?php

namespace MyNamespace;
function init(Hi $view)
{
}