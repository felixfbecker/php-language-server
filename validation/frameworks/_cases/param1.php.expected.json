{
    "references": {
        "MyNamespace\\Hi": [
            "./param1.php"
        ]
    },
    "definitions": {
        "MyNamespace": {
            "extends": [],
            "isGlobal": true,
            "isStatic": false,
            "canBeInstantiated": false,
            "symbolInformation": {
                "name": "MyNamespace",
                "kind": 3,
                "location": {
                    "uri": "./param1.php"
                },
                "containerName": ""
            },
            "type": null,
            "documentation": null
        },
        "MyNamespace\\init()": {
            "extends": [],
            "isGlobal": true,
            "isStatic": false,
            "canBeInstantiated": false,
            "symbolInformation": {
                "name": "init",
                "kind": 12,
                "location": {
                    "uri": "./param1.php"
                },
                "containerName": "MyNamespace"
            },
            "type__class": "phpDocumentor\\Reflection\\Types\\Mixed",
            "type": {},
            "documentation": null
        }
    }
}